go/mysql/replication_constants.go

```diff
@@ -115,6 +119,40 @@ const (
 	TypeGeometry = 255
 )
 
+var MySqlTypeToVitessType = map[int32]querypb.Type{
+	TypeDecimal:    querypb.Type_DECIMAL,
+	TypeTiny:       querypb.Type_INT8,
+	TypeShort:      querypb.Type_INT16,
+	TypeLong:       querypb.Type_INT32,
+	TypeFloat:      querypb.Type_FLOAT32,
+	TypeDouble:     querypb.Type_FLOAT64,
+	TypeNull:       querypb.Type_NULL_TYPE,
+	TypeTimestamp:  querypb.Type_TIMESTAMP,
+	TypeLongLong:   querypb.Type_INT64,
+	TypeInt24:      querypb.Type_INT24,
+	TypeDate:       querypb.Type_DATE,
+	TypeTime:       querypb.Type_TIME,
+	TypeDateTime:   querypb.Type_DATETIME,
+	TypeYear:       querypb.Type_YEAR,
+	TypeNewDate:    querypb.Type_DATETIME,
+	TypeVarchar:    querypb.Type_VARCHAR,
+	TypeBit:        querypb.Type_BIT,
+	TypeTimestamp2: querypb.Type_TIMESTAMP,
+	TypeDateTime2:  querypb.Type_DATETIME,
+	TypeTime2:      querypb.Type_TIME,
+	TypeJSON:       querypb.Type_JSON,
+	TypeNewDecimal: querypb.Type_DECIMAL,
+	TypeEnum:       querypb.Type_ENUM,
+	TypeSet:        querypb.Type_SET,
+	TypeTinyBlob:   querypb.Type_BLOB,
```

Comment:
are these mapping correct?

Reply:
You can just use `sqltypes.MySQLToType`: https://github.com/vitessio/vitess/blob/master/go/sqltypes/type.go#L240